build.gradle

```diff
@@ -78,13 +77,16 @@ ext {
     releaseVersionCode = 34
     releaseVersionName = '1.5.7'
 
-    androidCompileSdkVersion = 28
-    androidBuildToolsVersion = '28.0.3'
-    androidXVersion = '1.0.0'
-    minSdkVersion = 14
-    targetSdkVersion = 28
+    androidCompileSdkVersion = 29
+    androidBuildToolsVersion = '29.0.2'
+    minSdkVersion = 15
```

Comment:
Is there really anything stop we from using 14 as `minSdkVersion`?

Reply:
No, I raised it to try and move to newer dependencies and I just left it to match the sample app to 15, since api 14 is disappeared from the graph.
I can move back, tell me what you think.
We should consider moving it to 17 in the future for android-gif-drawable, to update to the ndk r18.

Reply:
I think we should stay at api 14, even if GlideImageViewFactory need api 17, we shouldn't bump all modules to 17, which will affect some library users which don't need that.

Reply:
moved back to 14 for now